kiali/kiali
# Apps lists page optimized to query per cluster

### Describe the change

Apps page was querying backend per namespace, now it queries per cluster by providing selected namespaces.
New API endpoing "api/clusters/apps" is added.

### Steps to test the PR

Create a set of namespaces:
for i in {1..200}; do kubectl create ns test-$i; done
Open the Apps list page and select all apps.
Compare the loading time between before and now.
Check if all fields are loaded correctly.

### Automation testing

Modified e2e and unit tests.

### Issue reference
https://github.com/kiali/kiali/issues/4832
